adapters/repos/db/vector/hnsw/startup.go

```diff
@@ -118,7 +118,7 @@ func (h *hnsw) restoreFromDisk() error {
 	// make sure the visited list pool fits the current size
 	h.pools.visitedLists.Destroy()
 	h.pools.visitedLists = nil
-	h.pools.visitedLists = visited.NewPool(1, len(h.nodes)+500)
+	h.pools.visitedLists = visited.NewPool(1, len(h.nodes)+512)
```

Comment:
if you would put that value in `const` in `maintanance.go` file then you would be able just reference it here

Reply:
same reasoning as above